src/app/modules/table2/table-cell/table-cell.component.scss

```diff
@@ -0,0 +1,13 @@
+.table--button {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: 100%;
+  @media screen and (max-width: 1080px) {
+    max-width: 300px;
+  }
+}
+
+.table_cell.table_cell_checkbox label span {
```

Comment:
you should be able to use the class `sr-only` on any element to hide it from view.  Can you add that to the table cell component for checkboxes instead of doing this here? There are some different styles associated with `sr-only` that have worked well for us in the past